Mathlib/Algebra/Polynomial/Eval/Defs.lean

```diff
@@ -451,15 +436,11 @@ theorem C_mul_comp : (C a * p).comp r = C a * p.comp r := by
 theorem natCast_mul_comp {n : ℕ} : ((n : R[X]) * p).comp r = n * p.comp r := by
   rw [← C_eq_natCast, C_mul_comp]
 
-@[deprecated (since := "2024-04-17")]
-alias nat_cast_mul_comp := natCast_mul_comp
```

Comment:
Another blank line should be removed here and also in what looks like many other places.